packages/core/src/shaderlib/pbr/pbr_helper.glsl

```diff
@@ -8,7 +8,7 @@ float computeSpecularOcclusion(float ambientOcclusion, float roughness, float do
 float getAARoughnessFactor(vec3 normal) {
     #ifdef HAS_DERIVATIVES
         vec3 dxy = max( abs(dFdx(normal)), abs(dFdy(normal)) );
-        return max( max(dxy.x, dxy.y), dxy.z );
+        return 0.04 + max( max(dxy.x, dxy.y), dxy.z );
     #else
         return 0.04;
     #endif
```

Comment:
Do you have some paper or dissertation about SpecularAA?

Reply:
can refers to https://www.jcgt.org/published/0010/02/02/paper.pdf
![image](https://user-images.githubusercontent.com/17639043/165878186-85e13416-6ce4-4eea-b019-bf6c9dc08b70.png)

simplified in https://yusuketokuyoshi.com/papers/2017/Error%20Reduction%20and%20Simplification%20for%20Shading%20Anti-Aliasing.pdf
![image](https://user-images.githubusercontent.com/17639043/165878331-97fb5cfe-9e34-4c7f-a8da-e174fc97fd3b.png)

In addition, the method of the paper is to map the roughness, our engine uses the perceptual roughness, so we avoid the square root operation by assume that ` THRESHOLD ` is `0`, `SCREEN SPACE VARIANCE` is `0.5` to avoid square `dudv`